ownCloud/Client/Actions/MoreViewHeader.swift

```diff
@@ -25,17 +25,19 @@ class MoreViewHeader: UIView {
 	private var titleLabel: UILabel
 	private var detailLabel: UILabel
 	private var favoriteButton: UIButton
+	private var showsIcon : Bool = true
 
 	var thumbnailSize = CGSize(width: 60, height: 60)
 	let favoriteSize = CGSize(width: 24, height: 24)
 
 	var showFavoriteButton: Bool
+	var adaptBackgroundColor : Bool
 
 	var item: OCItem
 	weak var core: OCCore?
 	var url: URL?
 
-	init(for item: OCItem, with core: OCCore, favorite: Bool = true) {
+	init(for item: OCItem, with core: OCCore, favorite: Bool = true, adaptBackgroundColor adaptBgCol: Bool = false) {
```

Comment:
@felix-schwarz I think `_ adaptBackgroundColor: Boo`l would be nicer

Reply:
I agree `adaptBgCol` is ugly, but I'd like to keep the label. Addressed it by removing `adaptBgCol`.